w-books/src/App/screens/BookDetail/index.js

```diff
@@ -1,19 +1,14 @@
 import React from 'react';
-import {
-  BrowserRouter as Router,
-  Route,
-  Link
-} from 'react-router-dom'
+import { Link } from 'react-router-dom'
 import './styles.css';
-import Header from '../../components/Header/index.js';
 import Comments from  './components/Comments/index.js';
 import Information from  './components/Information/index.js';
 import Sugestions from  './components/Sugestions/index.js';
 import BookList from '../../../assets/dummy-books.json';
 import CommentList from '../../../assets/dummy-comments.json';
 
 function BookDetail({ match }) {
-  let book = BookList.find((element) => { return element.id == match.params.bookId})
+  let book = BookList.find((element) => { return element.id === match.params.bookId})
```

Comment:
again, move code to index.js and ´layout.js`. Do it for all views.

So, do getSuggestions and BookList.find((element) => { return element.id == match.params.bookId}) in componentWillMount and store data in state, then pass data to child

Reply:
Done!